### PR TITLE
standby for nondashboard ucr db

### PR DESCRIPTION
**Pull request by sravfeyn**
@snopoke

### CODE REVIEW

File: environments/icds-cas/known_hosts [hunk 1]
```diff
@@ -319,3 +319,4 @@
 |1|upPQw7Vsg6EikyyDrIVi3BJ38Ps=|ZWL+tU/rC92NgTnU2W/DwLqMshQ= ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBMHAhALvcIS35zalZMN0dr1ruqhxRnniB4p1x/D+Y/Ar0bRc9SnPfPJs6qOI9+m2bjNTTQreSFBf9P/0ynv+89E=
 |1|uvB50QaQ//ElTwKzZJ10QiZlRjk=|/KTUkTZAjx1SlZZpiPKSf1aX6cM= ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBJ15cNqlfP0DGpMPjYc6GOlAZcicSJWf15aybVd9D9HeCcvCuHVZ8NdDSOGIAWeOgnnoesIl6ZeMv/Jt5DY2sVE=
 |1|vc92V97gGCzIEaDtt6Km+QlwEJI=|9NDEP50K5aVTlx1Oeo8jts44JDE= ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBIY/tkKSumIa3yaWBwYRVzQakWRfMgZjS1Zb9ZgeVuQkp/v5+Kc1A1/e8quDm/L8Vav7DR5nd13ZQ/vshYe6Qjs=
+100.71.184.70 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCvpx2HwY7OKFbtUcIH9yoA43MyVQZ+x+a2Rumn+4B5vbBkgyp8CFMnvO+unoMgfEErNAeuslid4dqISm5LFnXn2OX8Jj//XElIpQyNWauSykVwjdzhCF3RLzFPCihkOIT4lZzqnaYbQopn0K8DqxeflzJ3hDh9tr1Sm06b0jCqtQpAqnCcaIGLLOZAgF9ziYz6lT0B9YZ3dlVdJ77fs2HYfTpxkXjOdfDoN/UgeeU6kxmonGsdliin7G6XWGxQKBs5vft6v4sr7ZvFCyUKIyS7iVZIydX9ubgaYlEb84n/CQAjYGG7/tey0etPPVtRrtNqdCPMTucQTPAioAAZ9KI3
```
Comment (snopoke) [inline environments/icds-cas/known_hosts]: was this intentional?

Comment (sravfeyn) [reply environments/icds-cas/known_hosts]: Yeah?

Comment (snopoke) [reply environments/icds-cas/known_hosts]: looks like it was added by ssh instead of using the 'update-local-known-hosts'

Comment (sravfeyn) [reply environments/icds-cas/known_hosts]: Ha, that's right. Will update

**Pull request merged**